src/main/java/com/gistpetition/api/answer/domain/AnswerRepository.java

```diff
@@ -4,12 +4,15 @@
 import org.springframework.data.repository.history.RevisionRepository;
 import org.springframework.stereotype.Repository;
 
+import java.util.List;
 import java.util.Optional;
 
 @Repository
 public interface AnswerRepository extends RevisionRepository<Answer, Long, Long>, JpaRepository<Answer, Long> {
     Optional<Answer> findByPetitionId(Long petitionId);
 
+    List<Answer> findListByPetitionId(Long petitionId);
```

Comment:
```suggestion
    List<Answer> findAllByPetitionId(Long petitionId);
```
`List->All`로 수정하는 것은 어떨까요?